src/Operation.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Aidantwoods\SecureHeaders;
+
+interface Operation
+{
+    /**
+     * Transform the given set of headers
+     *
+     * @param HeaderBag $headers
+     * @return void
+     */
+    public function modify(HeaderBag $headers);
```

Comment:
Just for clarity here, shall we include the an ampersand to make clear that the object passed can be modified 😉 

As in changing to `public function modify(HeaderBag &$headers);`

(The PHP [docs lie](https://php.net/manual/functions.arguments.php) and say that arguments are passed by value, so the fact that `$headers` object can be (and is) modified (in the outer scope) might easily be overlooked)